docs/integrations/security-threat-detection/netskope.md

```diff
@@ -99,18 +94,35 @@ to_user, app_session_id  nodrop
 | count
 ```
 
-## Installing the Netskope app
+## Collection configuration and app installation
+
+import CollectionConfiguration from '../../reuse/apps/collection-configuration.md';
+
+<CollectionConfiguration/>
+
+:::important
+Use the [Cloud-to-Cloud Integration for Netskope](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-source) to create the source and use the same source category while installing the app. By following these steps, you can ensure that your Netskope app is properly integrated and configured to collect and analyze your Netskope data.
+:::
+
+### Create a new collector and install the app
```

Comment:
Can we divide the three section similar to Kubernetes Non Kubernetes tab. since other wise it becomes a very long doc page and user will be following only one of them depending upon their use case.

Reply:
Thats a good idea, but the only drawback with that is, user will not be able to jump easily to the required section from the start of page. He would be need to come to the "Collection configuration section" then select the required tabs for collection. Now with this heading format, he can easily navigate through the page TOC in the left side of doc. 
To add on, having such long title for tabs will not be an ideal choice.